test/modules/toolbox/components/shareToolContent/ShareToolDialog.test.js

```diff
@@ -44,33 +50,39 @@ describe('ShareToolDialog', () => {
 		expect(element.shadowRoot.querySelectorAll('.share_item').length).toBe(1);
 	});
 
-	it('copies the url to the clipboard, when click', async (done) => {
-		const copySpy = spyOn(shareServiceMock, 'copyToClipboard').and.callFake(() => Promise.resolve());
+	it('copies the https url to the clipboard, when click', async (done) => {
+		const URL = 'https://mock.url';
```

Comment:
Please follow the name convention for local variables  => `url` (same in line 71)